leobot_web_server/web_content/index.html

```diff
@@ -19,7 +19,7 @@ <h1>OPEN SOURCE TELEPRESENCE ROBOT</h1>
         <div class="wrapper">
             <section class="video">
                 <h3 class="video-header">VIDEOSTREAMING FROM CAMERA 1</h3>
-                <iframe class="video-streaming" src="http://res.cloudinary.com/dvhn8lvvz/video/upload/c_fill,g_east,h_504,w_671/v1515307396/Localization_and_Mapping_SLAM_on_ROS_zbqkjf.mp4" frameborder="0" gesture="media" allow="encrypted-media" allowfullscreen></iframe>
+                <iframe class="video-streaming" src="http://0.0.0.0:8090/stream?topic=/leobot/stereocamera/left/image_raw&width=640&height=470" frameborder="0" gesture="media" allow="encrypted-media" allowfullscreen></iframe>
```

Comment:
In case if someone will pass another port for media web server this code will fail